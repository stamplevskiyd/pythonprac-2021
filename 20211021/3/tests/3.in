1
a,b$c%d^e(fGh)*76<q :w

