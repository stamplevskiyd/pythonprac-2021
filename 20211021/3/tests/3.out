a b c d e q w 